spec/lib/batch/batch_metadata_updates.rb

```diff
@@ -0,0 +1,102 @@
+require 'rails_helper'
+require "#{Rails.root}/lib/batch/batch_metadata_updates_654"
+
+RSpec.describe 'rails runner lib/batch/batch_metadata_updates_654.rb' do
+  before do 
+    @user = FactoryGirl.create(:user)
```

Comment:
Isn't this FactoryBot now?

Reply:
It is, but the gem isn't updated in this app.